frontend/public/components/nav/admin-nav.tsx

```diff
@@ -85,6 +85,7 @@ const MonitoringNavSection = connect(monitoringNavSectionStateToProps)(Monitorin
 const AdminNav = () => (
   <React.Fragment>
     <NavSection title="Home">
+      <HrefLink href="/dashboards" name="Dashboards" />
```

Comment:
I believe we need an `activePath`

Reply:
This will need an RBAC check so it only shows up for users who have access. `required={FLAGS.CAN_LIST_NS}` is probably sufficient.